components/trusted-app-mgt/org.wso2.carbon.identity.trusted.app.mgt/pom.xml

```
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024, WSO2 Inc. (http://www.wso2.com).
```

```suggestion
  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

check the license header

---

check in other relevant places as well.